setup.sh

```diff
@@ -67,14 +67,13 @@ function setupSwap() {
 
 function installSSH() {
     echo "--------------------------------------------"
-    echo "Installing SSH..."
-    sudo apt-get install openssh-client
+    # echo "Installing SSH..."
+    # sudo apt-get install openssh-client
     read -p "Generating SSH keys. Enter email address to be associated: " email
```

Comment:
It would make sense to try using git's global user email. If it is not present, then ask for the email.